### PR TITLE
Making a note about issues being in the main repo

### PR DESCRIPTION
**Pull request by bobstrecansky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +29,6 @@ $ composer require open-telemetry/opentelemetry-php-contrib
 
 Please, consult the documentation of the main [OpenTelemetry php project](https://github.com/open-telemetry/opentelemetry-php).
 
+## Issues
+
+Issues have been disabled for this repo in order to help to not segment issues between this and the main opentelemetry-php repo.  If you have an issue you'd like to raise about this issue, please use the [OpenTelemetry PHP Issue section](https://github.com/open-telemetry/opentelemetry-php/issues/new/choose).  Please prefix the title of the issue with [opentelemetry-php-contrib].
```
Comment (brettmc) [inline README.md:34]: fix wording: "issue you'd like to raise about this repository" ?

**Review by brettmc — APPROVED** [review]
approved, fixing wording optional

File: README.md (hunk 1 above)
Comment (tidal) [reply README.md:34]: I also feel a lot of non native speaker would stumble upon the double infinitive use.
Maybe "in order to help not segmenting ..." would be better.

**Comment by bobstrecansky** [comment]
Closed in favor of https://github.com/open-telemetry/opentelemetry-php-contrib/pull/42.  I did the web edit thing for this repo, but now I'm not able to edit this branch because of the protected repo status 👎🏼

**Pull request closed without merging**